R/label-dollar.R

```diff
@@ -14,6 +14,11 @@
 #'   value is less than `largest_with_cents` which by default is 100,000.
 #' @param prefix,suffix Symbols to display before and after value.
 #' @param negative_parens Display negative using parentheses?
+#' @param billion_scale Named list indicating suffixes given to large values
```

Comment:
Could this be `rescale` or something a bit more generic?

Reply:
Hi @hadley - I renamed the argument to `rescale_large` to be more generic, while still giving some clue as to when it would be used. I also made some other adjustments so it plays nicely with the existing `accuracy` and `scale` arguments.

I think this is ready for another review, when you have time. 👍